distributionviewer/core/static/js/app/components/containers/chart-container.js

```diff
@@ -44,16 +46,16 @@ class ChartContainer extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.showOutliers !== prevProps.showOutliers && this.props.metric.type === 'numeric') {
-      this.activeData = this.props.showOutliers ? this.allData : this.dataExcludingOutliers;
+    if (this.props.selectedScale !== prevProps.selectedScale || this.props.showOutliers !== prevProps.showOutliers) {
```

Comment:
This whole block is a little unreadable. We don't have to make them functions but we can probably store some of the booleans in their own var.